src/pt_br/index.js

```diff
@@ -727,3 +815,37 @@ export const i19TransactionsType = {
   payment: 'Pagamento',
   recurrence: 'Assinatura'
 }
+
+export const i19TriggersAction = {
```

Comment:
```suggestion
export const i19ApiActions = {
```

Pode ser comum em outros casos também (é do REST da verdade), então acho que isto não deveria se limitar a `triggers`...